pmoves/services/model-registry/main.py

```
@@ -0,0 +1,511 @@
+                error_detail = response.json().get("message", error_detail)
+            except:
+                pass
+            raise HTTPException(status_code=response.status_code, detail=error_detail)
+
+        return response.json()
+
+    async def get_active_models(self) -> List[Dict]:
+        """Get all active models with provider details."""
+        return await self._request("GET", "v_active_models")
+
+    async def get_service_models(self, service: str, function: str = None) -> List[Dict]:
```

Ruff (0.14.13)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)